### PR TITLE
Run Composer Dependency Analyser in CI

### PR DESCRIPTION
**Pull request by szepeviktor**
Let's see the output!

### CODE REVIEW

**Comment by szepeviktor** [comment]
✗ composer-dependency-analyser Tool composer-dependency-analyser is not supported

https://github.com/shivammathur/setup-php/issues/897

File: .github/workflows/phpstan.yml [hunk 1]
```diff
@@ -21,11 +21,11 @@ jobs:
       - uses: actions/checkout@v4
 
       - name: Setup PHP
-        uses: shivammathur/setup-php@v2
+        uses: shivammathur/setup-php@v4
```
Comment (szepeviktor) [inline .github/workflows/phpstan.yml:24]: 💣

**Comment by szepeviktor** [comment]
image

Ignore results related to tests/ because dev deps are removed.
So actually `illuminate/support` is missing.

**Comment by xHeaven** [comment]
Thanks! I've added an exclusion for tests so that it doesn't kill the CI unnecessarily.

**Pull request merged**